src/HPND.xml

```diff
@@ -29,20 +29,20 @@
          <optional>and</optional> that both <optional>that</optional> 
          <optional>the</optional> copyright notice 
          and this permission notice appear in supporting documentation<optional>, and that the name <optional>of</optional> 
-            <alt match=".*">&lt;copyright holder&gt;</alt> 
+            <alt match=".*" name="copyrightholder0">&lt;copyright holder&gt;</alt> 
```

Comment:
Can we camelCase there names?  Or hyphenate them, or whatever, as long as we're consistent?  severalwordswithoutseparators is hard to read ;).